templates/parts/mini-cart.html

```diff
@@ -31,13 +31,15 @@
 
 		<!-- wp:buttons -->
 		<div class="wp-block-buttons">
-			<!-- wp:button {"align":"center","className":"is-style-outline"} -->
-			<div class="wp-block-button aligncenter is-style-outline">
-				<a href="/" class="wp-block-button__link">Start shopping</a>
-			</div>
+			<!-- wp:button -->
```

Comment:
@gigitux Thanks for taking care of this! I think instead of using a fake button to load the core button style, we should add dedicated style for the Shopping Button block.

Reply:
The problem that I see is that every theme has a different style for the button element. In this way, we are able to have the theme style for the button.

Reply:
@gigitux If the `Shopping Button` block is removed then added back, the button style will be broken again because the button block markup is removed.

Reply:
This is also breaking the Site Editor:

https://user-images.githubusercontent.com/5423135/153164658-cf690320-f883-4b58-9e62-87fc7393df8c.mov

Reply:
@dinhtungdu Yes! :( 

How can we style the button to make it consistent with most themes? Do you have any idea?

Reply:
I think we should keep this block simple with a basic style (square outlined button with color inherit from the parent). If users need more customization, they can replace this block by a button block.

Reply:
I can continue working on this PR if you don't mind : )

Reply:
Yes, sure!

Reply:
I updated the PR, please take another look!